src/transformState.js

```diff
@@ -5,7 +5,29 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+
+  Object.assign(state, state);
+
+  for (const action in actions) {
+    switch (actions[action].type) {
```

Comment:
use destructuring

Reply:
I cannot understand how to use destruction here properly